src/screens/Sidebar/Blocking.js

```diff
@@ -1,46 +1,50 @@
 import React from 'react';
-import { View, Image, TouchableOpacity, FlatList, TextInput } from 'react-native';
+import {
+  View,
+  Image,
+  TouchableOpacity,
+  FlatList,
+  Keyboard,
+  Text,
+  ActivityIndicator,
+} from 'react-native';
 import { connect } from 'react-redux';
-import { Text, Container, Content, Left, Right, Item } from 'native-base';
+import { Left, Right } from 'native-base';
+import { InstantSearch } from 'react-instantsearch/native';
+import { connectInfiniteHits } from 'react-instantsearch/connectors';
 import PropTypes from 'prop-types';
 import * as colors from 'kitsu/constants/colors';
-import menu from 'kitsu/assets/img/tabbar_icons/menu.png';
+import SearchBox from 'kitsu/components/SearchBox';
+import { Kitsu, setToken } from 'kitsu/config/api';
+import { kitsuConfig } from 'kitsu/config/env';
+import defaultAvatar from 'kitsu/assets/img/default_avatar.png';
+import { SidebarTitle, ItemSeparator } from './common/';
+import styles from './styles';
 
-import { SidebarHeader, SidebarTitle, ItemSeparator } from './common/';
-
-class Blocking extends React.Component {
-  static navigationOptions = ({ navigation }) => ({
-    header: () => <SidebarHeader navigation={navigation} headerTitle={'Blocking'} />,
-    tabBarIcon: ({ tintColor }) => (
-      <Image source={menu} style={{ tintColor, width: 20, height: 21 }} />
-    ),
-  });
-
-  onUnblockUser = (user) => {
-    // TODO: handle button press.
-  };
-
-  renderItem = ({ item }) => (
-    <Item style={styles.sectionListItem}>
+const RowItem = ({ type, item, onPress }) => {
```

Comment:
can we go with the pattern of `renderRowItem` instead of a capital letter? I know they're technically components but lets reserve `PascalCase` components for ones that get exported. (I think there's an example of where I did the opposite of what I'm asking and I've got it fixed in a future PR :D)